mne/viz/evoked.py

```diff
@@ -1472,7 +1472,11 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
     ch_types = list(set(channel_type(example.info, pick_)
                     for pick_ in picks))
     # XXX: could possibly be refactored; plot_joint is doing a similar thing
-    if any([type_ not in _DATA_CH_TYPES_SPLIT for type_ in ch_types]):
+    # Valid data types ordered for consistency (same as _plot_evoked)
+    valid_channel_types = ['eeg', 'grad', 'mag', 'seeg', 'eog', 'ecg', 'emg',
+                           'dipole', 'gof', 'bio', 'ecog', 'hbo', 'hbr',
+                           'misc']
```

Comment:
Sorry for the delay @cmmoenne. Yeah I would say plot whatever is available if possible. I think that's what @jaeilepp might have been getting at in terms of the `_VALID_CHANNEL_TYPES`, a list of channel types that we support across plotting functions. Although maybe that would just be all of them, in which case we already have such a list.

Is that what you were thinking @jaeilepp?